### ISSUE TITLE
How do you generate a swagger yaml file instead of json?

### ISSUE THREAD

**Issue by sboagibm** [grpc-ecosystem/grpc-gateway#467]
It's not clear to me how to generate a yaml file, instead of a json file.  Is that possible, or do I need a converter?

**Comment by fmovlex** [comment]
Seems like the json format is the only output available.  
The [online editor](https://editor.swagger.io/) offers a quick JSON to YAML conversion though.

**Comment by achew22** [comment]
Can you elaborate on your need for a YAML file instead of JSON? I believe anything that reads YAML should read JSON.

**Comment by sboagibm** [comment]
It's been a while since I made this issue, and I've since moved on.  My recollection is that our overall swagger file is YAML, and I wanted to embed the swagger generation from the grpc-gateway tool into the larger rest api, and the JSON was problematic for that purpose.  The pattern I'm working with is a rest api proxying to multiple gRPC services. I was hoping to do some sort of auto-embedding, but have since given up on the notion, at least for now.
